Screen.py

```diff
@@ -0,0 +1,206 @@
+import pygame
+import colors
+from teams import Team
+import os
+
+pygame.init()
+
+SCREEN_WIDTH = 480
+SCREEN_HEIGHT = 680
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+squares = []
+PICTURES_PATH = 'pictures'
+
+BOARD_LINE = 8
+HEIGHT_OF_SCOREBOARD = 200
+SCORE_BOARD = pygame.Surface((screen.get_width(), HEIGHT_OF_SCOREBOARD))
+FONT = pygame.font.SysFont('comicsansms', 30)
+
+
+class Square:
+    WIDTH = 60
+    HEIGHT = 60
+
+    def __init__(self, x, y, color, tur, line):
+        self.rect = pygame.Rect(x, y, self.WIDTH, self.HEIGHT)
+        self.color = color
+        self.original_color = color
+        self.tur_cord = tur
+        self.line_cord = line
+        self.current_piece = None
+
+    def draw(self):
+        pygame.draw.rect(screen, self.color, self.rect)
+
+    def coloring_square_by_original_color(self):
+        if self.color == self.original_color:
+            if self.original_color == colors.DARK_BROWN:
+                self.color = colors.DARK_RED
+            else:
+                self.color = colors.LIGHT_RED
+        else:
+            self.color = self.original_color
+
+    def __str__(self):
+        return f'(line: {self.line_cord}, tur: {self.tur_cord})'
+
+
+def square_is_valid(tur, line, is_white_team):
+    """
+    Check if square is in board bounds and not taken by teammate piece.
+    """
+    if 0 <= line < BOARD_LINE:
+        if 0 <= tur < BOARD_LINE:
+            check_square_piece = squares[line][tur].current_piece
+            if check_square_piece is not None:
+                # Check if other piece is on the same team.
+                return is_white_team != check_square_piece.is_in_white_team
+            # Next move is inside board and empty square.
+            return True
+    return False
+
+
+def draw_bg(team_got_turn: Team, team_doesnt_got_turn: Team):
+    white_timer = team_doesnt_got_turn.timer
+    black_timer = team_got_turn.timer
+    if team_got_turn.is_white_team:
+        white_timer = team_got_turn.timer
+        black_timer = team_doesnt_got_turn.timer
+
+    screen.blit(SCORE_BOARD, (0, 0))
+    bg_image = pygame.image.load(os.path.join(PICTURES_PATH, 'boardscore_bg.png'))
+    SCORE_BOARD.blit(bg_image, (0, 0))
+
+    draw_squares_bg()
+    draw_who_turn_is(team_got_turn)
+    draw_timer(white_timer, black_timer)
+    draw_score(team_got_turn, team_doesnt_got_turn)
+
+
+def draw_squares_bg():
+    for line in squares:
+        for square in line:
+            square.draw()
+
+
+def draw_who_turn_is(team_got_turn):
+    if team_got_turn.is_white_team:
+        text = FONT.render('White Player Turn', False, colors.WHITE)
+    else:
+        text = FONT.render('Black Player Turn', False, colors.BLACK)
+
+    SCORE_BOARD.blit(text, (SCORE_BOARD.get_width() / 2 - 80, 0))
+
+
+def draw_timer(white_timer, black_timer):
+    minutes = white_timer.get_minutes_left()
```

Comment:
Currently, this function is actually the same code twice.
If you can do it a nice way, change this function to draw_timers, that calls twice draw_timer (a new function) that receives only one timer and draw it.